internal/watchdtui/watchdtui_test.go

```diff
@@ -387,33 +382,23 @@ func TestInteractiveInput(t *testing.T) {
 				fmt.Println(out)
 			}
 
-			// Update golden file
-			if update {
-				t.Logf("updating golden file %s", goldPath)
-				err = os.WriteFile(goldPath, []byte(normalizeOutput(t, out)), 0600)
-				require.NoError(t, err, "Cannot write golden file")
-			}
+			got := normalizeOutput(t, out)
 
-			want, err := os.ReadFile(goldPath)
-			require.NoError(t, err, "Cannot load golden file")
+			goldPath := filepath.Join(absPath, testutils.GoldenPath(t))
```

Comment:
No, because we changed directory earlier in the function and this would be bad. Believe me, I made this mistake earlier 🥲

Reply:
Mind adding a comment on the assignement then telling we are capturing the current path here because we are changing the directory afterwards? Because I can really forsee some quick cleanup which will do exactly that :p
(maybe we should then rename as it’s not absPath anymore, it’s absPackageDirPath or something like this)